examples/error_messages.rs

```diff
@@ -10,7 +10,6 @@
 
 use auto_impl::auto_impl;
 
-
```

Comment:
All of these double-empty-lines were on purpose. I'm not sure if rustfmt did this automatically, but could you remove your "2-empty-lines -> 1-empty-line" changes?

Reply:
Sure